packages/components/src/Modal/ModalSnapshots.test.tsx

```diff
@@ -15,23 +15,22 @@ describe("Button Variations", () => {
   });
 
   test("modal shows with primary learning button", () => {
-    const tree = renderer
-      .create(
-        <Modal
-          title="Teaching Modal"
-          open={true}
-          primaryAction={{
-            label: "I Would Like to Know More",
-            variation: "learning",
-          }}
-          secondaryAction={{
-            label: "Nevermind",
-          }}
-        >
-          Learn a lesson?
-        </Modal>,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+    const { container } = render(
+      <Modal
+        title="Teaching Modal"
+        open={true}
+        primaryAction={{
+          label: "I Would Like to Know More",
+          variation: "learning",
+        }}
+        secondaryAction={{
+          label: "Nevermind",
+        }}
+      >
+        Learn a lesson?
+      </Modal>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
avoiding that div eh? 😆

Reply:
I think I did it just to test and forgot it 🤦 updating...